.github/ISSUE_TEMPLATE/alpha_feature.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Alpha Feature Requirements
+about: Requirements for promoting a feature to alpha
+---
+
+This page lists the requirements for promoting a feature to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Requires approved RFC in order to merge. 
+
+**Config**
+
+- [ ] Requires explicit user action to enable (e.g. a config field, config resource, or installation action). 
```

Comment:
Maybe we should have them describe how the feature is enabled here rather than just a checkbox?

Reply:
A link to documented steps on how to enable it. :)

Reply:
Added a field to fill in/link to the instructions.